### PR TITLE
Fix get_budget response type and update examples to properly format balance

### PR DESCRIPTION
**Pull request by bsiaotickchong**

### CODE REVIEW

**Comment by bsiaotickchong** [comment]
* **#184** <a href="https://app.graphite.dev/github/pr/uma-universal-money-address/uma-js-sdk/184?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @bsiaotickchong and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: examples/react/src/components/Balance.tsx [hunk 1]
```diff
@@ -1,38 +1,51 @@
...
   }, [nwcRequester]);
 
+  let balanceString = "";
+  if (error) {
+    balanceString = error;
+  }
+  if (!balance || isFetching) {
+    balanceString = "Loading...";
+  } else if (!balance.currency) {
+    balanceString = `${balance.balance} msat`;
```
Comment (jklein24) [inline examples/react/src/components/Balance.tsx]: We should probably show SATs and divide by 1000 in this case. mSATs are tiny.

File: examples/vanilla/src/main.ts [hunk 2]
```diff
@@ -41,7 +41,20 @@ const fetchBalance = async (requester: NwcRequester) => {
   balanceElement.innerText = `Balance: Loading...`;
   try {
     const response = await requester.getBalance();
-    balanceElement.innerText = `Balance: ${response.balance}`;
+    let balanceString = "";
+    if (!response.currency) {
+      balanceString = `${response.balance} msat`;
```
Comment (jklein24) [inline examples/vanilla/src/main.ts]: Same here.

File: examples/react/src/components/Balance.tsx (hunk 1 above)
Comment (bsiaotickchong) [reply examples/react/src/components/Balance.tsx]: updated!

File: examples/vanilla/src/main.ts (hunk 2 above)
Comment (bsiaotickchong) [reply examples/vanilla/src/main.ts]: updated!

**Review by jklein24 — APPROVED** [review]

**Comment by bsiaotickchong** [comment]
### Merge activity

* **Oct 10, 4:33 PM EDT**: A user merged this pull request with [Graphite](https://app.graphite.dev/github/pr/uma-universal-money-address/uma-js-sdk/184).

**Pull request merged**